.circleci/config.yml

```diff
@@ -616,17 +618,18 @@ jobs:
       image_tag:
         description: 'docker image tag to use'
         type: string
-      make:
-        description: 'make target'
+      schedule:
+        description: 'test schedule'
         type: string
-        default: check-minimal
+        default: ''
       test:
-        description: 'the test that should be run multiple times'
+        description: 'the test file path that should be run multiple times'
         type: string
+        default: ''
       runs:
         description: 'number of times that the test should be run in total'
         type: integer
-        default: 1600
+        default: 50
```

Comment:
50 seems a bit too low to detect flakyness, especially given that we're running with 32 parallism. I think 8 runs per tests should always be fine. So, let's do 8*32 = 256